### ISSUE TITLE
Front end fixes for splash page

### ISSUE THREAD

**Issue by jazmynlatimer** [codeforamerica/typeseam#54]
- [x] add bg-img to .above_the_fold section (in img folder)
- [x] add list style checkmarks for .opportunities ul, before the list item
- [x] center btn in .apply_backup section
- [x] add cfa logo to footer (in img folder)
- [x] full width background-colors for all sections

**Comment by bengolder** [comment]
@jazmynlatimer I took care of all of these, but only minimally.

Regarding the check marks, I'm not sure if the checkmarks I used will suffice. I added a unicode check mark to each list item.
There are a couple things I could do to get closer to the type of checkmark you used in the comps.
* add a circle behind the existing unicode checkmark
* add a font icon library to the code to have access to a more similar check mark (might be overkill if we're only using one symbol).

Here's a preview of the current state:
![screen shot 2016-03-20 at 5 03 08 pm](https://cloud.githubusercontent.com/assets/451510/13908365/5e098a38-eebe-11e5-8e60-a04c0c40b01f.png)
